data-prepper-api/src/main/java/org/opensearch/dataprepper/model/metric/JacksonMetric.java

```diff
@@ -25,7 +25,7 @@
 public abstract class JacksonMetric extends JacksonEvent implements Metric {
 
     protected static final String NAME_KEY = "name";
-    protected static final String SCOPE_KEY = "scope";
+    protected static final String SCOPE_KEY = "instrumentationScope";
```

Comment:
This affects both standard and OpenSearch. Is this what we want?

Reply:
It is not used in Opensearch mode. It already has other keys defined like this 
```
    static final String INSTRUMENTATION_SCOPE_NAME = "instrumentationScope.name";
    static final String INSTRUMENTATION_SCOPE_VERSION = "instrumentationScope.version";
    static final String INSTRUMENTATION_SCOPE_ATTRIBUTES = "instrumentationScope.attributes";
```
The new SCOPE_KEY with `instrumentationScope` is only used in the standard mode.